folderTypes.cfg

```diff
@@ -19,8 +17,7 @@ Slice:
     name: "slice"
     info:
         slice quality: "list", ["1", "2", "3", "4", "5"]
-        location: "string"
-        orientation: "string"
+        surface: "list", ['medial', 'lateral']
```

Comment:
I'm wondering if this should just be 'Flipped' or 'Not flipped' to be consistent with LIMS? We can recover whether it's the medial or lateral surface from the hemisphere info above I think